utilities/tools/Test-NamePrefixAvailability.ps1

```diff
@@ -32,16 +27,26 @@ Test-NamePrefixAvailability @inputObject
 
 Test if namePrefix 'carml' is available.
 #>
+$inputObject = @{
+    NamePrefix = 'jpe01'
+    Tokens     = @{
+        Location          = 'westeurope'
+        ResourceGroupName = 'validation-rg'
+        SubscriptionId    = '00000000-0000-0000-0000-000000000000'
+        ManagementGroupId = '00000000-0000-0000-0000-000000000000'
+        RemoveDeployment  = $false
+        deploymentSpId    = '00000000-0000-0000-0000-000000000000'
+    }
+}
+Test-NamePrefixAvailability @inputObject
```

Comment:
Please remove this test. Also, I recommend to usually put it AFTER the function as you'll otherwise run the old code ;)